__tests__/basic-auth-middleware.test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+require('supergoose');
+const middleware = require('../src/auth/middleware/basic.js');
+const Users = require('../src/auth/models/users.js');
+
```

Comment:
replace 'supergoose' with '@code-fellows/supergoose' ,and take out the 'auth' part